README.md

```diff
@@ -490,6 +490,85 @@ To quickly copy the value of a variable into the clipboard, use the keyboard pre
 
 **Note**: Values set with `verb-var` and `verb-set-var` will be lost if the buffer is killed.
 
+### Verb Variables from External Files
+
+To further keep sensitive information safe and separate from Verb `.org` files, Verb variables can also be loaded from either JSON or Emacs Lisp external configuration files. Use the `Verb-Prelude` Org mode property followed by the name of the external configuration file to load. Any file is loaded and applied as a prelude before the request being generated and sent.
+
+**Note**: Files that are GPG or EasyPG encrypted can opened and decrypted automatically by Emacs if configured appropriately. See: [Emacs Auth-source manual](https://www.gnu.org/software/emacs/manual/auth.html) for more information.
+
+The `Verb-Prelude` property may be set at the buffer level (very top of the Org document) or various header levels, and every heading level in the hierarchy with the `Verb-Prelude` property will also be loaded. If the same setting within an environment configuration file is specified at a parent and a child level, then the child will override the parent. This allows more specific or different settings to be done for a lower-level request than at a higher-level. All other file unique variables would be additive to the collection of Verb variables.
+
+Below is an example of configuring external configuration files at top buffer level, and then a different file for *Foobar Development* at lower level 2.
+
+``` org 
+#+Verb-Prelude: /path/to/prod-foobar-env.el.gpg
+
+* Foobar Blog API                    :verb:
+
+template https://foobar-blog-api.org/api/v1
+Accept: application/json
+
+** Foobar Production
+
+get /status
+
+** Foobar Development
+:properties:
+:Verb-Prelude: /path/to/dev-foobar-env.el.gpg
+:end:
+
+template https://foobar-blog-api.org/api/v2
+Accept: application/json
+
+get /status
```

Comment:
Hmm, this request spec doesn't look like it would work the intended way, we are defining two requests (one with `template` and another one with `get`) under the same heading 🤔

Reply:
Good eye!  Had extra template in there.